internal/commands/create.go

```diff
@@ -224,6 +226,62 @@ func getRegoFilePaths(path string) ([]string, error) {
 	return regoFilePaths, nil
 }
 
+func parsePolicies(policyPaths []string, libraryPaths []string) ([]*regoPolicy, error) {
+	policies, err := loadPolicies(policyPaths)
+	if err != nil {
+		return nil, err
+	}
+	libraries, err := loadPolicies(libraryPaths)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, p := range policies {
+		if len(p.policy.Imports) > 0 {
```

Comment:
nit: to reduce nesting, we can check if the policy doesn't have any imports, and if not, `continue`